src/components/home/homemidsection/projects.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "name": "NAKO",
+    "description": "The German National Cohort (“NAKO Gesundheitsstudie”) is an interdisciplinary, population-based cohort study that will follow the long-term medical histories of 200,000 participants over 25-30 years. As Germany’s largest health study, the overarching aim of the National Cohort is to inform more effective disease prevention strategies, with a focus on seven major disease groups: cancer, diabetes, and cardiovascular, neurologic and psychiatric, infectious, respiratory and musculoskeletal diseases. It will provide a major, central resource for population-based epidemiology in Germany, and will help to identify new and tailored strategies for early detection, prediction, and primary prevention of major diseases.",
+    "chart_location": "",
+    "chart_alt": "Chart",
+    "learn_more_href": "#"
+  },
+  {
+    "name": "NAKO",
```

Comment:
Do we not have another project? we are repeating the same.

Reply:
I have no clue. I need to be given that information. This is just as a test that mutliple projects work.